src/info.rs

```diff
@@ -439,9 +440,11 @@ impl Info {
         let mut remote_url = String::new();
         let mut repository_name = String::new();
 
+        let remote_regex = Regex::new(r"remote.[a-zA-Z0-9]+.url").unwrap();
```

Comment:
```suggestion
        let remote_regex = Regex::new(r"remote\.[a-zA-Z0-9]+\.url").unwrap();
```
`.` means any character in regex. Did you mean to use `\.`?

Reply:
You're absolutely right! the `.` is not meant to be a regex pattern